bundle/manifests/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -231,6 +231,8 @@ spec:
                               type: object
                             type: array
                         type: object
+                      disableIngressCreation:
```

Comment:
Can you add a description?

Reply:
changed it to not be a boolean and added description